### ISSUE TITLE
Setup of development environment issues amd README.md

### ISSUE THREAD

**Issue by mark6w** [dixon1e/smoker-beacon#1]
Not sure if I should enter these each as one or all together, but we can always separate them out as needed.

Question:  What is N.B.?  Not Built?
Question:  What is TL;DR

I'm assuming that this is all boilerplate, and there's some extra boilerplate to be added, specifically ./dispatcher.py, ./sb.py, ./images/logo.png, ./startall.sh true?

I don't get the point of item #9 changing the appearance of the desktop, and not sure what flavor of *nux it's referencing, as I don't have anything resembling any of that (I just rt-click on the desktop for desktop appearance changes anyway).

**Comment by dixon1e** [comment]
N.B. == Nota Bene == Take Note == Pay Attention!
TL;DR == Too Long; Didn't Read == Here's the only thing you care about

Yes, the boilerplate code has not been added.

#9 is cosmetic and not needed

The Linux in question is called Linaro, it targets ARM devices.

**Comment by mark6w** [comment]
hrm, so do I need to build a VM with linaro?
